src/emulate.c

```diff
@@ -290,6 +287,7 @@ static bool emulate(riscv_t *rv, rv_insn_t *ir)
             rv->X[ir->rd] = pc + ir->insn_len;
         /* check instruction misaligned */
         if (insn_is_misaligned(rv->PC)) {
+            rv->compressed = (ir->insn_len == INSN_16);
```

Comment:
It should be `rv->compressed = false;` since instruction `jal` is always uncompressed. Therefore, we just assign an exact value.

Reply:
Done.